package/scripts/params.py

```diff
@@ -71,7 +71,7 @@
 tmp_kafka_hosts = ''
 for i, val in enumerate(kafka_hosts):
   tmp_kafka_hosts += val + ':' + kafka_bind_port
-  if (i + 1) < len(tmp_kafka_hosts):
+  if i < len(tmp_kafka_hosts):
```

Comment:
assuming python has map and join functions, that would be much cleaner here.  array.map (to append port to each), then join(',')